backend/open_webui/utils/chat.py

```
@@ -66,7 +47,9 @@ async def generate_direct_chat_completion(
     user: Any,
     models: dict,
 ):
-    log.info("generate_direct_chat_completion")
+    import time
```

Ruff (0.12.2)

Redefinition of unused `time` from line 7

Remove definition: `time`

(F811)